comment/utils.py

```diff
@@ -88,6 +101,11 @@ def get_comment_context_data(request, model_object=None, comments_per_page=0):
 
     allowed_flags = getattr(settings, 'COMMENT_FLAGS_ALLOWED', 0)
     oauth = request.POST.get('oauth') or request.GET.get('oauth')
+    if oauth == 'true':
+        oauth = True
+    else:
+        oauth = False
```

Comment:
You are right. I would suggest the following:
```suggestion
    if oauth and oauth.upper() == 'TRUE':
        oauth = True
    else:
        oauth = False
```